tests/integration/test_specs/chord_err.yaml

```diff
@@ -0,0 +1,37 @@
+description:
+    name: chord_err
+    description: test the chord err problem
+
+env:
+  variables:
+    OUTPUT_PATH: ./studies
+
+study:
+- name: process_samples
+  description: Process samples. Purposefully try to trigger the chord err
+  run:
+    cmd: |
+        if [ $(SAMPLE) == "RAISE" ];
+        then
+          exit $(MERLIN_RAISE_ERROR)
+        else
+          echo "Success for sample $(SAMPLE)"
+        fi
+- name: step_2
```

Comment:
I think it’d be good to add another step as a dependency on step 1 but before the join step so we test chords of chains (like have 3 steps). We should also have a test with parameters and samples together

```
1-2
      \
        3
      /
1-2
```

Reply:
maybe step 2 can utilize parameters and samples together and then step 3 can join everything like step 2 is currently doing? I'll work on that